src/index.html

```diff
@@ -1,17 +1,255 @@
+                name="password"
+              >
+            </label>
+          </div>
+        </fieldset>
+        <fieldset>
+          <legend>
+            An interesting fact about you!
+          </legend>
+          <div 
+            class="div-class"
+          >
+            Do you love cats?
+            <label>
+              <input 
+                type="radio" 
+                name="love_cats"
+              >
+              Yes
+            </label>
+            <label>
+              <input 
+                type="radio" 
+                name="love_cats"
+              >
+              No
+            </label>
+          </div>
+          <div  
+            class="div-class"
+          >
+            <label>
+              What is your favorite color?
+              <input 
+                type="color" 
+                name="color"
+              >
+            </label>
+          </div>
+          <div 
+            class="div-class"
+          >
+            <label>
+              What time do you go to bed?
+              <input 
+                type="time" 
+                name="time" 
+                min="00:00:00" 
+                max="24:00:00"
+                size="3" 
+                step="1"
+              >
+            </label>
+          </div>
+          <div 
+            class="div-class"
+          >
+            <label>
+              What are your favorite brand of cars?
+              <select 
+                name="select" 
+                id="select"
+                multiple size="4"
+              >
+                <option 
+                  value="BMW"
+                >
+                  BMW
+                </option>
+                <option 
+                  value="Audi"
+                >
+                  Audi
+                </option>
+                <option 
+                  value="Lada"
+                >
+                  Lada
+                </option>
+              </select>
+            </label>
+          </div>
+          <div>
+            <label>
+              How do you rate our work?
+              <input 
+                type="range" 
+                name="range" 
+                value="0"
+              >
+            </label>
+          </div>
+        </fieldset>
+        <fieldset>
+          <legend>
+            Additional info
+          </legend>
+          <div 
+            class="div-class">
+            <label>
+              Comments:
+              <textarea 
+                name="range">
+              </textarea>
+            </label>
+          </div>
+          <div>
+            <label>
+              Would you recommend us?
+              <select 
+                name="selectWoild" 
+                id="selectWoild"
+              >
+                <option 
+                  value="Yes"
+                >
+                  Yes
+                </option>
+                <option 
+                  value="No"
+                >
```

Comment:
```suggestion
                <option value="No">
```
For all tags with just one attribute, you can place it in one line.
Could you check your HTML markup and correct it everywhere?

Reply:
pay attention to the comment

Reply:
Не, не могу, должно быть всё одинаково, чтобы привыкать к высокому.